Scicloud (#273)
ubvu/open-handbook: topics/scicloud.qmd

---
title: SciCloud
---

> IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network so services running on it can be made internet accessible. It can also run on an internal network for easy security and access to some VU services.

Reviewer: Just a though, maybe somewhere in the very beginning a short sentence explaining what it is "for dummies"? Like an example of a situation when you might need it?

Author: I've added "What can it be used for?" and added some typical use cases.

Purchase of virtual server capacity takes place according to standard building blocks, combinations of the following two options can be purchased: 

• Building Block 1: 1 CPU core with 2 GB RAM 

• Building Block 2: 1 CPU core with 4 GB RAM 
<!-- add a link to page on costs somewhere (? haven't found it yet) --> 

Different configurations are possible in consultation and based on availability.  
 
You are responsible for installing and maintaining any applications on the server yourself. On a consultancy basis, ITvO can assist in the configuration and optimization of the virtual server.

## SciCloud functionalities 
- A virtual server with guaranteed CPU power and reserved RAM.
- Full administrator (root) access, so that the server can be adapted to your own wishes.
- A choice of templates with Windows and Linux operating systems. It is also possible to install your own OS, license and security permitting.
- Linking to shares on SciStor is possible.
- Monitoring of operation and used capacity of the virtual server through a web portal.
- Transparent cost, you pay a fixed monthly fee based on the configuration via internal billing.

## Storage 

> By default, the virtual server is provided with 30 GB of storage capacity. Do you need more storage? Then a link with [SciStor](./scistor.qmd) storage is a possibile providing access to fast networked storage. ITvO can advise on this.

Reviewer 2: ```suggestion
By default, the virtual server is provided with 30 GB of storage capacity. Do you need more storage? Then a link with [SciStor](./scistor.qmd) storage is a possibility providing access to fast networked storage. ITvO can advise on this.
```

## Backup 
It is possible to take _snapshots_ or _clones_ of the virtual server yourself. This allows you to go back to a previously created configuration, if necessary. In addition, ITvO makes daily backups with which a virtual machine can be restored up to a month ago on request. 

> ## Requesting process

Reviewer 2: ```suggestion
## Requesting access
```

> Via form on [ServiceNow](https://services.vu.nl/esc), go to: IT > My work field > Research > SciCloud > Order SciCloud capacity

Reviewer 2: Would it be useful to add another section 'Contact'? The ServiceNow form is mentioned for requesting access, but perhaps it's good to explicitly explain how researchers can get in touch with ITvO if they have questions about SciCloud (if they just want more information, or if they encounter problems when using it), referring both to the ITvO email address and the possibility to ask a question through ServiceNow (I guess just with the general IT contact form)? What do you think, @peer35?

Author: As discussed I put in a mailto link for now.

---
title: SciCloud
---
## What is it?

> IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network so services running on it can be made internet accessible. It can also run on an internal network for extra security and access to some VU services.

Reviewer 2: ```suggestion
IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network, so services running on it can be made internet accessible. It can also run on an internal network for extra security and access to some VU services.
```

SciCloud is based on the open source middleware [OpenNebula (ONE)](https://opennebula.io/) and offers a service that allows users to quickly start with a virtual server based on available templates, or to upload one of their own virtual images. This concept is known as Infrastructure as a Service (IaaS).

## What can it be used for?
In principle you can run any software you want on a SciCloud server. Linux OS is preferred, but Windows server is also a possibility.

Typical use cases are:

> - Running web applications and services that need to be publicly accessible 24/7. For example a website showcasing of your research, a wiki, chatbot, etc.

Reviewer 2: ```suggestion
- Running web applications and services that need to be publicly accessible 24/7, for example a website showcasing of your research, a wiki, chatbot, etc.
```

> - Hosting a licensed server application for your research group. Such as software for analysis, software needed to run lab equipment or elab journal software.

Reviewer 2: ```suggestion
- Hosting a licensed server application for your research group, such as software for analysis, software needed to run lab equipment or elab journal software.
```

- The Virtual Server can also be connected to [SciStor](./scistor.qmd) for environments that need fast access to large amounts of data. 

Note that the SciCloud environment has no GPU, so it is not suitable for running graphical environments and machine learning. 

## How to request access

> Via form on 🔒[ServiceNow](https://services.vu.nl/esc), go to: IT > My work field > Research > SciCloud > Order SciCloud capacity

Reviewer 2: ```suggestion
Via a form on 🔒[ServiceNow](https://services.vu.nl/esc), go to: IT > My work field > Research > SciCloud > Order SciCloud capacity
```

After you have requested a SciCloud virtual server, IT for Research will schedule an interview to discuss your wishes. 

## Are there costs involved?
Configurations of a virtual server are based on standard "Building Blocks", combinations of the following two options can be purchased: 

• Building Block 1: 1 CPU core with 2 GB RAM 

• Building Block 2: 1 CPU core with 4 GB RAM 

Costs are calculated as: €1 per CPU per month, €1 per GB memory per month, €0.40 per GB storage per year (minimum 20GB)

## Getting started

> The IT for Research Team will setup the server(s) with the OS of your choice (Windows or Linux) and will set up a root account for you:

Reviewer 2: ```suggestion
The IT for Research Team will set up the server(s) with the OS of your choice (Windows or Linux) and will set up a root account for you:
```

> - For access to a Linux server you will need to provide a personal SSH key protected with a passphrase. You will be able to access SSH fromn home. 

Reviewer 2: ```suggestion
- For access to a Linux server you will need to provide a personal SSH key protected with a passphrase. You will be able to access SSH from home. 
```

- For access to Windows servers use Remote Desktop Protocol (RDP), only via 🔒[eduVPN institute access](https://vu.eduvpn.nl).

> Root access to the server allows you to install akk the software you need. Note that IT for Research can help you get started and will automatically install OS security updates, but you are responsible for installing and maintaining software yourself.

Reviewer 2: ```suggestion
Root access to the server allows you to install all the software you need. Note that IT for Research can help you getting started and will automatically install OS security updates, but you are responsible for installing and maintaining software yourself.
```
